E2E/helpers/customSteps.helper.ts

```diff
@@ -11,6 +11,13 @@ class CustomHelper extends Helper {
     await Playwright.fillField(locator, value);
   }
 
+  async grabNumberFrom(string: string) {
+    const { Playwright } = this.helpers;
+    string = await Playwright.grabTextFrom(string);
```

Comment:
Why would you assign grabTextFrom result to a string, which is passed as a parameter and represents completely different variable?